src/commands/copyRelativePathToClipboard.ts

```diff
@@ -0,0 +1,26 @@
+import { env, TextEditor, Uri } from 'vscode';
+import { Commands } from '../constants';
+import type { Container } from '../container';
+import { command } from '../system/command';
+import { ActiveEditorCommand, getCommandUri } from './base';
+
+@command()
+export class CopyRelativePathToClipboardCommand extends ActiveEditorCommand {
+	constructor(private readonly container: Container) {
+		super(Commands.CopyRelativePathToClipboard);
+	}
+
+	async execute(editor?: TextEditor, uri?: Uri) {
+		uri = getCommandUri(uri, editor);
+		const repoPath = this.container.git.getBestRepository(editor)?.path;
```

Comment:
Using the `getRelativePath` below, get the `.uri` prop rather than path here. Basically want to transition to using Uris everywhere as possible.